index.Rmd

```diff
@@ -181,6 +181,7 @@ plotRGB(image2, r = 4, g = 3, b = 2, axes = TRUE,
         stretch = "lin", main = "True Color Composite")
 box(col="white")
 ```
+
```

Comment:
I think adding an extra line here will make it so ###FALSE COLOR doesn't show up next to the image.